### PR TITLE
Newer version of the italian translation (it.json)

### PR DESCRIPTION
**Pull request by kz-n**
Moved meta to the top, added me to the translators too, updated Plugins and Themes to be present, removed the module store as not present in the english file.

### CODE REVIEW

File: langs/it.json [hunk 1]
```diff
@@ -1,3 +1,7 @@
+{
```
Comment (CanadaHonk) [inline langs/it.json:1]: This JSON is invalid, the meta should be inside the main object.

**Review by CanadaHonk — CHANGES_REQUESTED** [review]

File: langs/it.json [hunk 2]
```diff
@@ -38,7 +42,4 @@
     }
   },
```
Comment (CanadaHonk) [inline langs/it.json]: Invalid JSON because of this trailing comma here as well.

**Review by CanadaHonk — CHANGES_REQUESTED** [review]

File: langs/it.json (hunk 2 above)
Comment (kz-n) [reply langs/it.json]: Sorry, tried to do this with the web github editor and got confused

**Comment by CanadaHonk** [comment]
Looks good now, thank you!

**Pull request merged**